### PR TITLE
[Doc] [NextJS] Walkthrough: Deploying to Vercel

### PR DESCRIPTION
**Pull request by illiakovalenko**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update (non-breaking change; modified files are limited to the `/docs` directory)

## Checklist:

- [ ] I have read the Contributing guide.
- [ ] My code follows the code style of this project.
- [ ] My code/comments/docs fully adhere to the Code of Conduct.
- [ ] My change is a code change and it requires an update to the documentation.
- [ ] My change is a documentation change and it requires an update to the navigation.

### CODE REVIEW

**Review by AncaIO — APPROVED** [review]

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md [hunk 1]
```diff
@@ -3,4 +3,35 @@ name: vercel
...
+
+Using the `publish:end` webhook, you can automatically trigger static site generation, cache clearing, and other automated activities with webhook support.
+
+To enable the `Publish:end` hook, you must: 
+
+1. [Create a deploy hook](https://vercel.com/docs/more/deploy-hooks#creating-a-deploy-hook) in Vercel.
+1. On your Sitecore instance, edit `/App_Config/Sitecore/JavaScriptServices/Sitecore.JavaScriptServices.AppServices.PublishingWebHook.config.example`:
+	* `url` - Required. The URL of the hook to be invoked.
+	* `method` - Optional. The HTTP method for invoking webhook. Possible values are `POST` or `GET`. The default method is `POST`. 
+	* `site` - Optional. The sites which should trigger the webhook when published. If you provide the site parameter, the webhook will be invoked if the published item root is an ancestor, descendant, or equal to the configured site's root item.
```
Comment (ambrauer) [inline docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md:34]: What is the default action? Will it be triggered for any Sitecore publish (all sites)?

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md [hunk 2]
```diff
@@ -3,4 +3,35 @@ name: vercel
...
+
+Vercel is an all-in-one platform with Global CDN supporting static & Jamstack deployment. Vercel has first-class support for Next.js.
+
+Using Vercel, you can easily deploy the sample app to production. 
+
+We assume you have set up the Next.js sample application to connect to a Sitecore instance for the following steps.
+
+To deploy the Next.js sample app to production: 
+1. Execute the steps provided in the Next.js  guide for [Getting Started](https://nextjs.org/docs/deployment#getting-started) with deployment.
+2. In the root directory of the application, in the `.env` file, add or modify:
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md:17]: In the case that these are different for dev/prod, would these be defined in Vercel?

Comment (ambrauer) [inline docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md:17]: These would likely be added in Vercel directly (https://nextjs.org/docs/basic-features/environment-variables#environment-variables-on-vercel). We can mention they could be added to a .env file, but I think our recommendation should be adding in Vercel.

**Review by ambrauer — CHANGES_REQUESTED** [review]

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md [hunk 3]
```diff
@@ -3,4 +3,35 @@ name: vercel
...
+3. Push the changes to your Git provider.
+
+## `publish:end` webhook invocation
+
+Using the `publish:end` webhook, you can automatically trigger static site generation, cache clearing, and other automated activities with webhook support.
+
+To enable the `Publish:end` hook, you must: 
+
+1. [Create a deploy hook](https://vercel.com/docs/more/deploy-hooks#creating-a-deploy-hook) in Vercel.
+1. On your Sitecore instance, edit `/App_Config/Sitecore/JavaScriptServices/Sitecore.JavaScriptServices.AppServices.PublishingWebHook.config.example`:
```
Comment (nickwesselman) [inline docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md:31]: Can we change these steps working under the assumption that the developer will be creating a new configuration patch, and adding it to their VS solution, and deploying it from there? Renaming and editing files in the web root is not a real-world workflow for the platform.

**Review by nickwesselman — CHANGES_REQUESTED** [review]

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md (hunk 2 above)
Comment (nickwesselman) [reply docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md:17]: Dupe of Adam's comment.

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md (hunk 1 above)
Comment (illiakovalenko) [reply docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md]: @ambrauer Added default action

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md (hunk 2 above)
Comment (illiakovalenko) [reply docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md]: @ambrauer Changed this step

File: docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md (hunk 3 above)
Comment (illiakovalenko) [reply docs/data/routes/docs/nextjs/deploying-to-production/vercel/en.md]: @nickwesselman Changed this step, based on adding config to solution

**Review by ambrauer — APPROVED** [review]

**Review by nickwesselman — APPROVED** [review]

**Pull request merged**